Pocket/pocket-ios
# Signed Out Home

## Goal
This PR adds the **Signed Out Home** feature to Pocket. Specifically
* Adds the "Skip sign in" button in the onboarding screen, that allows users to access Pocket anonymously
* Tapping the button takes you to the signed out home, which contains recommendations and a banner that lets you sign in when you want. Tapping any Save action also prompts the user to sign in
* Opening a native collection and tapping any Save button, prompts the login screen
* Opening a syndicated article in the reader view and tapping any Saves button, prompts the login screen
* Tapping "Saves" in the tab bar, shows an empty saves list with a button that lets you sign in. Tapping filter buttons and the add url button also prompt the user to sign in
* Tapping "Settings" in the tab bar shows a settings page similar to the authenticated one, but the account section only contains a "Sign up or sign in" button.

> [!NOTE]
> at the moment, the exit survey that follows account deletion is disabled, since the UX has changed and now takes the user to signed out home after deletion. We can discuss a potential new UX that reinstates it.

## Test Steps
* Enable the new onboarding feature flag
* Build/run this branch, install from scratch
* Verify the steps described in the intro or showed in the video.
* Try multiple paths
* Test on real devices, both iPads and iPhones
* Delete, reinstall and verify that the original sign in button (now renamed from "Continue" to "Sign up or sign in") works as intended

## Related Issues
POCKET-10292
POCKET-10293
POCKET-10294 + child issues
POCKET-10295
POCKET-10297
POCKET-10298
POCKET-10300
POCKET-10318
POCKET-10375
POCKET-10401
POCKET-10413
POCKET-10414
POCKET-10415

## Video

<div align="center">
  <video src="https://github.com/user-attachments/assets/59be7307-1dff-4ae2-8f5f-5c4cb60b82d8"/>
</div>
